ida/data.yml

```diff
@@ -4921,7 +4921,9 @@ classes:
   Client::UI::Agent::AgentGatheringNote:
     vtbls:
      - ea: 0x141997360
-       base: Client::UI::Agent::AgentInterface
+       base: Client::UI::Agent::AgentGatheringNote
```

Comment:
This was correct, it should be AgentInterface as base

Reply:
Oh I see, because AgentGatheringNote doesn't inherit AgentInterface, makes sense. One moment.